### ISSUE TITLE
A11y - home page - dataset types

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1367]
## User story

- missing heading levels: heading goes from h1 to h4 directly
- non focusable interactive element: the + more item is not keyboard focusable because it is just a <li> element
- the +more button is nested inside a list and it's the only list item, take makes the list pointless
- low contrast red text in links
- The dataset links have an <img> tag followed by a text label and a number indicating the number of items. Make sure accessible name of links makes sense and does not present redundancies (i.e. announced category name once on focus, and number of items in a meaningful way)

![image](https://github.com/gigascience/gigadb-website/assets/143437854/b36e5d49-21cb-4126-8445-eced3cd2e4aa)

## Acceptance criteria

- [x] the +more interactive element should be keyboard focusable and be a button or have role button (prio 3)
- [x] do not skip heading levels, the "datatypes" heading should be a h2 (prio 3)
- [x] +more might not need to be a list since it's just one element (prio 1)
- [x] red text has enough contrast (prio 3)
- [x] a11y name for links (prio 1)

**Comment by luistoptal** [comment]
Working branch on https://github.com/luistoptal/gigadb-website/tree/new-feature/1367/home-page-dataset-types

**Comment by only1chunts** [comment]
is this work complete and merged? can we close this ticket?

**Comment by rija** [comment]
> is this work complete and merged? can we close this ticket?

Hi @only1chunts the issue has an associated PR #1438 (not yet reviewed)
 so we can close the ticket
